config/prod/mmason-general-bioc.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v9.j2
```

Comment:
You are using an old template. Update the template.

Reply:
we just cloned this